docs/root/configuration/listeners/listener_filters/connect_handler.rst

```diff
@@ -0,0 +1,36 @@
+.. _config_listener_filters_connect_handler:
+
+Connect Handler
+===============
+
+Connect Handler listener filter allows detecting whether the incoming request appears to be HTTP/1.x CONNECT, and if it is CONNECT, it removes the CONNECT by reading data from socket and responding with a 200 status code to indicate a connection to remote server is established. It also extracts remote server url from the CONNECT. This can be used to select a
```

Comment:
```suggestion
Connect Handler listener filter allows detecting whether the incoming request appears to be HTTP/1.x CONNECT, and if it is CONNECT, it removes the CONNECT by reading data from the socket and responding with a 200 status code to indicate a connection to the remote server is established. It also extracts the remote server url from the CONNECT. This can be used to select a
```